src/Raven.Server/Documents/Handlers/TimeSeriesHandler.cs

```diff
@@ -703,6 +703,24 @@ public async Task ConfigTimeSeries()
                 });
         }
 
+        private async Task<(long, object)> ModifyTimeSeriesConfiguration(TransactionOperationContext context, string name, BlittableJsonReaderObject configurationJson, string raftRequestId)
+        {
+            var configuration = JsonDeserializationCluster.TimeSeriesConfiguration(configurationJson);
+            configuration?.InitializeRollupAndRetention();
+            ServerStore.LicenseManager.AssertCanAddTimeSeriesRollupsAndRetention(configuration);
+            var editTimeSeries = new EditTimeSeriesConfigurationCommand(configuration, name, raftRequestId);
+            var result = await ServerStore.SendToLeaderAsync(editTimeSeries);
+
+            using (context.OpenReadTransaction())
+            {
+                var topology = ServerStore.Cluster.ReadDatabaseTopology(context, name);
+                var clusterTopology = ServerStore.GetClusterTopology(context);
+                await WaitForExecutionOnRelevantNodes(context, name, clusterTopology, topology.Members, result.Index);
```

Comment:
let's move this out of the `OpenReadTransaction` scope

Reply:
Done